bin/prep-release

```diff
@@ -8,40 +8,37 @@
 set -e
 
 if [ -z "$1" ]; then
-  echo "usage: bin/prep-release VERSION" >&2
+  printf "usage: bin/prep-release VERSION\n" >&2
   exit 64
 fi
 
-version=$1
-old_version=$(< VERSION)
-branch="release-$version"
+./bin/validate-release
 
-if ! make test; then
-  echo "test failure, not releasing" >&2
-  exit 1
-fi
+version="$1"
+old_version="$(< VERSION)"
+branch="release-$version"
 
 printf "RELEASE %s => %s\n" "$old_version" "$version"
-git checkout master
-git pull
+git fetch origin master
+git reset --hard origin/master
 
 git checkout -b "$branch"
 
 printf "%s\n" "$version" > VERSION
 make Gemfile.lock
 git add VERSION Gemfile.lock
-git commit -m "Release v$version"
+git commit --message "Release v$version"
 git push origin "$branch"
 
-branch_head=$(git rev-parse --short $branch)
+branch_head="$(git rev-parse --short "$branch")"
```

Comment:
Quotes are not needed for the left-hand side of an assignment, and we typically omit them.

```sh
branch_head=$(git rev-parse --short "$branch")
```